src/runner.sh

```diff
@@ -127,6 +127,7 @@ function runner::run_test() {
     "$function_name" "$data" 2>&1 1>&3
 
     runner::run_tear_down
+    runner::clean_mocks
```

Comment:
Where are the test?

Reply:
I added them to `tests/acceptance/mock_test.sh`

Reply:
we are also missing unit test of the function, no?

Reply:
Could you give me a hint how would you write unit tests for `runner::clear_mocks()`?